intermol/lammps/lammps_parser.py

```diff
@@ -175,6 +175,9 @@ def canonical_dihedral(self, params, dihedral, direction='into'):
             elif dihedral == FourierDihedral:
                 convertfunc = convert_dihedral_from_fourier_to_trig
                 converted_dihedral = TrigDihedral
+            elif dihedral == TrigDihedral:
+                convertfunc = lambda x: x
```

Comment:
Could you either remove the `converted_dihedral = dihedral  # Default` line above or move the `convertfunc` to the top as the base case and delete this branch?